packages/storybook/package.json

```
@@ -19,8 +19,8 @@
     "@mui/icons-material": "^5.10.2",
     "@mui/material": "^5.10.2",
     "@mui/x-data-grid": "5.17.1",
-    "@mui/x-data-grid-generator": "5.17.1",
-    "@mui/x-data-grid-premium": "5.17.1",
+    "@mui/x-data-grid-generator": "5.17.2",
+    "@mui/x-data-grid-premium": "5.17.2",
     "@mui/x-data-grid-pro": "5.17.1",
     "@mui/x-license-pro": "5.17.0",
```

Missing `grid` and `grid-pro` bumps.
```suggestion
    "@mui/x-data-grid": "5.17.2",
    "@mui/x-data-grid-generator": "5.17.2",
    "@mui/x-data-grid-premium": "5.17.2",
    "@mui/x-data-grid-pro": "5.17.2",
    "@mui/x-license-pro": "5.17.2",
```

---

We are not releasing packages with no change
Not sure if it's a good strategy, but it's how our Lerna is configured

---

But in changelog we are listing all the packages 🤔 
Don't we have a decision to try and maintain same versions for all the packages to reduce the headache for users?

---

I think the comes from the core where some package can go without updates for months and it would cause a lot of "useless" versions.

But to be honest I am not very up to date with this topic.
It almost never occurred before, except for the license package

Maybe @mnajdova, @DanailH or @oliviertassinari will know better.

---

+1 for not releasing `@mui/x-data-grid v5.17.2` and release `@mui/x-data-grid v5.17.3` next time if necessary

---

I think there is a real question for packages of the same "family"

Skipping releases on `@mui/x-license-pro` is one thing
Skipping releases on `@mui/x-data-grid` / `@mui/x-data-grid-pro` when we change only `@mui/x-data-grid-premium` is a little weird to me